### ISSUE TITLE
Postgis extension can't be installed in docker image

### ISSUE THREAD

**Issue by ethervoid** [CartoDB/bigmetadata#462]
Running the task `make diff-catalog` leads to this error

```
ERROR:  syntax error at or near ""
LINE 1: CREATE EXTENSION IF NOT EXISTS postgis
        ^
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/bigmetadata/tests/__init__.py", line 6, in <module>
    recreate_db('test')
  File "/bigmetadata/tests/util.py", line 32, in recreate_db
    dbname=dbname), shell=True)
  File "/usr/lib/python3.5/subprocess.py", line 626, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.5/subprocess.py", line 708, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command 'psql -d test -c "CREATE EXTENSION IF NOT EXISTS postgis"' returned non-zero exit status 1
Makefile:194: recipe for target 'diff-catalog' failed
```

**Comment by ethervoid** [comment]
After some investigation, I found that the culprit is the use of Dash `echo` shell command (even if we're using bash for it).

Checking the last version, 2.4, I see that it has no problem generating the SQL version of the extension and [I found why](https://github.com/postgis/postgis/commit/363a1d2ed3348f18901508d8948c74b9b65ac7cd#diff-f0386518d817c1ea8f98049720aef32c). We're using [this Docker image](https://github.com/kartoza/docker-postgis/blob/9.5-2.2/Dockerfile#L2) for our Postgresql + Postgis container which uses Debian as the base image. Debian uses [dash](https://es.wikipedia.org/wiki/Debian_Almquist_Shell) shell by default which has some problems with echo and backslashes character

In the 2.4 version echo was changed by `printf` but we're using 2.2.5 so we need to patch it to use `/bin/echo` instead the shell built-in command which is faulty
